### ISSUE TITLE
More example filters, especially FamilyFilter

### ISSUE THREAD

**Issue by shekenahglory** [falsecz/hbase-rpc-client#62]
Can you give more examples of syntax for adding filters?  I've implemented several, but for some reason I can't get the FamilyFilter to work (trying to filter by column family)

**Comment by wision** [comment]
Filters usage should be very similar to java client. All the filters can be found in proto file: https://github.com/falsecz/hbase-rpc-client/blob/master/proto/Filter.proto

**Comment by shekenahglory** [comment]
Thanks, I had seen this and got several filters to work correctly by following it, but I can't seem to get the FamilyFilter working.  This is my best guess: 
```     
list.addFilter({
  FamilyFilter: {
    compareFilter: {
      compareOp: 'EQUAL',
      comparator: {
        BinaryComparator: {
          comparable: 'f'
        }
      }
    }
  }
})
```
but I get this error `     Error: .Comparator#BinaryComparator is not a field: undefined
      at Message.set (node_modules/protobufjs/ProtoBuf.js:1736:35)` and I've tried a number of variations, usually with this sort of error or different errors.

**Comment by wision** [comment]
Fixed in #63
